core/src/test/java/cucumber/runtime/formatter/JSONFormatterTest.java

```diff
@@ -1241,7 +1241,7 @@ private String runFeaturesWithFormatter() {
         return report.toString();
     }
 
-    private Long milliSeconds(int milliSeconds) {
-        return milliSeconds * 1000000L;
+    private long milliSeconds(int milliSeconds) {
+        return milliSeconds;
```

Comment:
I think we can just use `Long.valueOf()` in the callers instead of this method now.